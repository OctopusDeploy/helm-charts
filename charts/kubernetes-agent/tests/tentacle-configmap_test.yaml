suite: "tentacle configuration"
templates:
  - templates/tentacle-configmap.yaml
tests:
  - it: "defaults to empty data (no data field)"
    asserts:
      - notExists:
          path: data

  - it: "when testing data is provided, populates the data field"
    set:
      testing:
        tentacle:
          configMap:
            data:
              key.1: "value 1"
              key.2: 123
    asserts:
      - equal:
          path: data["key.1"]
          value: "value 1"
      - equal:
          path: data["key.2"]
          value: 123
