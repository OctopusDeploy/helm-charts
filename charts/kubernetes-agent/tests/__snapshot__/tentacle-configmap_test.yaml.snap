should match snapshot:
  1: |
    apiVersion: v1
    kind: ConfigMap
    metadata:
      name: tentacle-config
      namespace: NAMESPACE
